packages/date-time-picker/src/date-time-picker.css

```diff
@@ -130,6 +130,8 @@ sp-popover .popover-content {
 .literal-segment {
     white-space: pre;
     user-select: none;
+
+    /* -webkit-user-select: none; */
```

Comment:
Should be deleted ?

Reply:
Uncommented [here](https://github.com/adobe/spectrum-web-components/pull/4962/files#diff-0f4bef552114d643584f8ef0a08fb745b8555778f6e4c90f3ac0275e161df866).